nixio/dimensions.py

```diff
@@ -333,43 +333,68 @@ def _create_new(cls, parent, index, data_frame, column):
         """
         newdim = super(DataFrameDimension, cls)._create_new(parent, index)
         newdim.data_frame = data_frame
-        newdim.column = column
+        newdim.column_idx = column
         newdim._dimension_type = DimensionType.DataFrame
         return newdim
 
     def get_unit(self, index=None):
+        """
+        Get the unit of a specific column in the referenced DataFrame
+        :param index: Index of the needed column
+        :type index: int
+
+        :return: Unit of the specified column
+        :rtype: str or None
+        """
         if index is None:
-            if self.column is None:
+            if self.column_idx is None:
                 raise ValueError("No default column index is set "
                                  "for this Dimension. Please supply one")
             else:
-                idx = self.column
+                idx = self.column_idx
         else:
             idx = index
         unit = None
         if self.data_frame.units is not None:
+
             unit = self.data_frame.units[idx]
         return unit
 
     def get_ticks(self, index=None):
+        """
+        Get the values in a specific column in the referenced DataFrame
```

Comment:
```suggestion
        Get the ticks of the Dimension from the referenced DataFrame.
        If an index is specified, it will return the values of the column
        in the referenced DataFrame at that index.
```